flexmeasures/data/migrations/versions/cb8df44ebda5_flexcontext_field.py

```diff
@@ -157,7 +157,9 @@ def upgrade():
 
         # Get fields-to-migrate from attributes
         market_id = attributes_data.get("market_id")
-        capacity_in_mw = attributes_data.get("capacity_in_mw")
+        capacity_in_mw = attributes_data.get("capacity_in_mw") or attributes_data.get(
+            "site_power_capacity"
+        )
```

Comment:
Isn't this still leaving a copy of each field in the attribute column? I think we want to move the fields rather than copy them. Should we use `pop` instead of `get`?

```suggestion
        capacity_in_mw = attributes_data.pop("capacity_in_mw", None) or attributes_data.pop(
            "site_power_capacity", None
        )
```

Reply:
Likewise for the other fields.

Reply:
(By the way, I like how your code prioritizes the `capacity_in_mw` value over the `site_power_capacity` value, in case they both exist and have different values, because `capacity_in_mw` would be the one actually in use.)

Reply:
the keys are already being popped at line 59 & 60